Mathlib/Topology/Spectral/Spectral.lean

```diff
@@ -0,0 +1,23 @@
+/-
```

Comment:
I was planning on calling this file `Basic` but `Spectral` is also fine.

Reply:
Probably `Basic` is better, yes